source/versions/filenames.rst

```diff
@@ -0,0 +1,121 @@
+always having to run an outdated Forge version in order to use SpongeForge.
+
+
+The format of the filename is:
+
+``spongeforge-<MCVersion>-<ForgeBuild>-<APIMajor>.<LatestAPIMinorRelease>.<RecommendedVersion(-RC<BuildNumber>)>.jar``
+
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``MCVersion``        | The Minecraft version. Only clients compatible with this version can connect.                 |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``ForgeBuild``       | Preferably your server should run this exact version of Forge (which can be identified in the |
+|                      | last part of Forge's version string).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``APIMajor``         | The major version of SpongeAPI implemented by this file (the ``X`` in                         |
+|                      | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``LatestAPI``        | The minor version of SpongeAPI implemented by this file (the ``Y`` in                         |
+| ``MinorRelease``     | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``Recommended``      | The released version of the implementation when **not** followed by ``-RC<BuildNumber>.``     |
+| ``Version``          | When this number is **not** zero, the SpongeAPI version in development is used (i.e. API      |
+|                      | Minor + 1).                                                                                   |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``-RC``              | The build number in development for the next implementation release. When a build number      |
+| ``<BuildNumber>``    | is present, the ``RecommendedVersion`` has **not** been released yet.                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+
+Examples
+~~~~~~~~
+
+SpongeForge Jar files will always follow this naming scheme to allow you to easily identify compatibility.
+
+For example, the file name ``spongeforge-1.12.2-2705-7.1.4.jar`` is compatible with Minecraft version ``1.12.2``, was 
+built with Forge ``14.23.4.2705`` (Build ``2705``), uses SpongeAPI ``7.2-SNAPSHOT``, is a recommended version, and is 
```

Comment:
> uses SpongeAPI ``7.2-SNAPSHOT`` 

should be

> guarantees compatibility with SpongeAPI ``7.1.0``

Reply:
Changed here and in the following example.